firefox_puppeteer/api/appinfo.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class AppInfo(BaseLib):
+
+    @property
+    def browserTabsRemoteAutostart(self):
```

Comment:
I'm mirroring the appinfo member, "browserTabsRemoteAutostart". I don't want to add another name that means the same thing.